aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -229,6 +229,52 @@ fn native_create_aggregator(
     ]))])
 }
 
+
+/***************************************************************************************************
+ * native fun create_unbounded_aggregator<IntElement: copy + drop>(): Aggregator<IntElement>;
+ **************************************************************************************************/
+
+ fn native_create_unbounded_aggregator(
+    context: &mut SafeNativeContext,
+    ty_args: Vec<Type>,
+    args: VecDeque<Value>,
+) -> SafeNativeResult<SmallVec<[Value; 1]>> {
+    if !context.aggregator_api_enabled() {
```

Comment:
this block (checking context for enabled) should be re-used in many methods, can we make it a method/macro call?